src/Pages/Login/Login.js

```diff
@@ -1,12 +1,82 @@
-import React, { Component } from "react";
+import React from "react";
+import "./Login.scss";
 
-class Login extends Component {
+class Login extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      id: "",
+      password: "",
+    };
   }
+
+  handleValue = e => {
+    const { id, value } = e.target;
+    this.setState({ [id]: value });
+  };
+
+  checkValidation = e => {
+    e.preventDefault();
+    const { id, password } = this.state;
+    const checkId = id.includes("@");
+    const checkPw = password.number;
+
+    if (checkId && checkPw) {
+      alert("로그인을 성공하셨습니다 😉 ");
+    }
+
+    if (!checkId) {
+      alert("아이디는 @ 를 포함해야 합니다");
+    }
+
+    if (!checkPw) {
+      alert("비밀번호가 틀립니다");
+    }
+
+    if (!checkId && checkPw) {
+      alert("로그인에 실패하셨습니다");
+    }
+  };
+
   render() {
-    return <></>;
+    return (
+      <div className="Login">
+        <div className="section-login">
+          <h1 className="title">로그인</h1>
+          <form>
+            <input
+              type="text"
+              placeholder="아이디를 입력해주세요"
+              id="id"
+              className="id box"
```

Comment:
id속성을 따로 사용하지 않고, className으로 id, password를 구분한뒤, 
아마 css 속성을 같이 주시려고 box className을 작성하신 것 같으신데 form 태그 안에 input 태그선택자로 nesting만 잘해도 "box"라는 className은 불필요합니다. 
```suggestion
              className="id"
```

Reply:
네 수정했습니다 👍